awx/main/tasks.py

```diff
@@ -1001,14 +1002,11 @@ def run(self, pk, isolated_host=None, **kwargs):
             try:
                 stdout_handle.flush()
                 stdout_handle.close()
-                # If stdout_handle was wrapped with event filter, log data
-                if hasattr(stdout_handle, '_event_ct'):
-                    logger.info('%s finished running, producing %s events.',
-                                instance.log_format, stdout_handle._event_ct)
-                else:
-                    logger.info('%s finished running', instance.log_format)
+                event_ct = getattr(stdout_handle, '_event_ct', 0)
```

Comment:
What actually sets this?

Reply:
Oh, the `_event_ct` predates this on the `OutputEventFilter`.  Disregard.

Reply:
Yes, this is a change we saw coming.

Personally, I have already found substantial value in the log message. There were all kinds of cluster debugging issue where I would search for this log, and then go in the API to `/jobs/N/job_events/` to compare to the number of events. When the instances are coming up/down unpredictably it's not surprising to _lose_ events. To the extent of me using that log, I've found it to be pretty reliable.

This change is automating that task and putting it in the API.